### ISSUE TITLE
data-breakpoint-em not functioning correctly anymore

### ISSUE THREAD

**Issue by fishonstilts** [mrjasonweaver/flexnav#158]
It appears that the ability to enter data breakpoints in ems is not working in the current version of Flexnav. When using ems the desktop view does not allow hover activity on the dropdown menus. Is there a way to re-enable this?

**Comment by mrjasonweaver** [comment]
@fishonstilts Hmm, the code hasn't changed since Nov 28, 2013. When did you last have ems working?

**Comment by fishonstilts** [comment]
I never had it working. I was trying to implement it since I saw that you had implemented it in early 2013. I changed the data-breakpoint to the data-breakpoint-em value and then the menus stopped working. I had downloaded the most recent of Flexnav zip and noticed that the Jquery code makes no mention of the "data-breakpoint-em" variable. I looked back at an earlier version from March 2013 on Github and noticed the presence of the "data-breakpoint-em" variable. Has it been lost somewhere along the way?
